src/networking.c

```diff
@@ -931,6 +931,31 @@ void addReplyHumanLongDouble(client *c, long double d) {
     }
 }
 
+static inline void _addReplyLongLongSharedHdr(client *c, long long ll, char prefix, robj **shared_hdr) {
```

Comment:
```suggestion
static inline void _addReplyLongLongSharedHdr(client *c, long long ll, char prefix,
                                              robj *shared_hdr[OBJ_SHARED_BULKHDR_LEN])
{
```
avoid passing a shared_hdr whose size is not `OBJ_SHARED_BULKHDR_LEN`.